samples/subsys/usb/hid-mouse/src/main.c

```diff
@@ -14,70 +14,61 @@
 #define LOG_LEVEL LOG_LEVEL_DBG
 LOG_MODULE_REGISTER(main);
 
-/* change this to use another GPIO port */
-#if DT_NODE_HAS_PROP(DT_ALIAS(sw0), gpios)
-#define PORT0 DT_GPIO_LABEL(DT_ALIAS(sw0), gpios)
-#else
-#error DT_GPIO_LABEL(DT_ALIAS(sw0), gpios) needs to be set
-#endif
+#define FLAGS_OR_ZERO(node)						\
```

Comment:
If we're not going to require that all controllers support a flags cell then perhaps this functionality should be lifted out to `DT_GPIO_FLAGS_OR_ZERO(node, gpios)`.  Not in this PR.

Reply:
> perhaps this functionality should be lifted out to DT_GPIO_FLAGS_OR_ZERO(node, gpios). Not in this PR.

Agreed on both counts